extensions/apns_message_handler.go

```diff
@@ -165,7 +173,7 @@ func (a *APNSMessageHandler) loadConfigurationDefaults() {
 // HandleResponses from apns.
 func (a *APNSMessageHandler) HandleResponses() {
 	for response := range a.PushQueue.ResponseChannel() {
-		a.handleAPNSResponse(response)
+		_ = a.handleAPNSResponse(response)
```

Comment:
Is there a reason why we are ignoring the return of this function?

Reply:
We were already ignoring. I added a `_` just so Goland would stop giving warnings. I'll add a log as well, we don't really have anything to do if that fails.